hawaii-international-conference-on-system-sciences-proceedings.csl

```diff
@@ -11,15 +12,22 @@
       <name>Jeff Hemsley</name>
       <email>jhemsley@uw.edu</email>
     </author>
+    <contributor>
+      <email>glink@unomaha.edu</email>
+      <name>Georg Link</name>
+    </contributor>
     <category citation-format="numeric"/>
     <category field="engineering"/>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2017-09-03T18:02:42+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <locale>
+    <style-options punctuation-in-quote="false"/>
+  </locale>
   <macro name="author">
-    <group suffix=".">
+    <group suffix=", ">
       <names variable="author">
-        <name initialize-with="." delimiter=", " and="text" name-as-sort-order="all" sort-separator=", "/>
+        <name and="text" delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="." name-as-sort-order="first"/>
```

Comment:
Do we have anything more than the templace? One of the examples corresponds to `name-as-sort-order="first"`, the other `="never"`

Reply:
Unfortunately, HICSS does not have a good style guide and when I pull examples from last year the authors follow different conventions and get published. The option reflects what I believe to look the best, based on previous years, while still adhering to the limited style guide.

Reply:
PS: I have a conditional accept with these settings and no comment was made about changing the style.